### ISSUE TITLE
Syntax error

### ISSUE THREAD

**Issue by erictwo** [vedetta-com/dithematic#7]
# zoneadd example.com

Imported TSIG key tsig.example.com hmac-sha512
reconfig start, read /etc/nsd.conf
/etc/nsd.conf:2: at 'control-enable:': error: syntax error
read /etc/nsd.conf failed: 1 errors in configuration file

# ssh -4 -i /home/tsig/.ssh/id_ed25519 -l tsig dig.example.com "exit"

Warning: Permanently added '111.222.333.444' (ECDSA) to the list of known hosts.
PTY allocation request failed on channel 0
/usr/local/share/doc/dithematic/validate.tsig:6: error: syntax error
/usr/local/share/doc/dithematic/validate.tsig:6: error: last key has no secret blob
read /usr/local/share/doc/dithematic/validate.tsig failed: 2 errors in configuration file
Connection to 111.222.333.444 closed

# env NS="dig.example.com" tsig-share tsig.example.com

Warning: Permanently added '104.156.237.135' (ECDSA) to the list of known hosts.
Imported TSIG key tsig.example.com hmac-sha512
reconfig start, read /etc/nsd.conf
/etc/nsd.conf:20: error: cannot open include file '/etc/tsig.example.com': Permission denied
/etc/nsd.conf:22: at 'remote-control:': error: syntax error
/etc/nsd.conf:22: error: last key has no secret blob
read /etc/nsd.conf failed: 3 errors in configuration file

I do not see these syntax errors  was hoping you might know what they are.
Thanks!

**Comment by horia** [comment]
Could be a wrong permission for "tsig.example.com"
```console
dot# ls -l /var/nsd/etc/tsig.example.com
-rw-r-----  1 root  _nsd  -   88B Sep  7 03:31 /var/nsd/etc/tsig.example.com
```

... which is created by
```console
dot# tsig-secret tsig.example.com
```

... and changed in nsd and pdns configurations by
```console
dot# tsig-change tsig.example.com
```

> /etc/nsd.conf:20: error: cannot open include file '/etc/tsig.example.com': Permission denied

I have `secret: include: /var/nsd/etc/tsig.example.com` in "nsd.conf"

"tsig.example.com" should be initialized by the Makefile, maybe something went wrong during install.

**Comment by erictwo** [comment]
You are correct it stops with errors a couple of times and I am only able to finish up the Makefile install manually which obviously is whats causing these problems. I will try again see if I can get a little more info to work with and check permissions on /var/nsd/etc/tsig.example.com. Thanks for you time Horia.
